README.adoc

```diff
@@ -0,0 +1,29 @@
+= Brine
```

Comment:
Switch README to AsciiDoc to be consistent with the current technology for the rest of the docs in this project.